app/models/user.py

```diff
@@ -17,6 +19,55 @@ class User(baseModel):
 
     def __str__(self):
         return str(self.__dict__)
+    
+    def __init__(self,*args, **kwargs):
+        super().__init__(*args,**kwargs)
+        self._ldsCache = None  
+
+    @property
+    def isLaborDepartmentStudent(self):
+        if self._ldsCache is not None:  
+            return self._ldsCache
+            
+        if not self.student:
+            self._ldsCache = False
+            return self._ldsCache
+
+        from app.models.laborStatusForm import LaborStatusForm
+        from app.models.formHistory import FormHistory
+        today = date.today()
+
+        # Subquery to check if student has an approved release form
+        released_forms = (
+            FormHistory
+            .select(FormHistory.formID)
+            .where(
+                (FormHistory.releaseForm.is_null(False)) &  # Has a release form
+                (FormHistory.status == "Approved")     # Release form is approved
+            )
+        )
+
+        labor_status_forms = (
+            LaborStatusForm
+            .select()
+            .join(Department, on=(LaborStatusForm.department == Department.departmentID))
+            .join(FormHistory, on=(
+            (FormHistory.formID == LaborStatusForm.laborStatusFormID) &
+            (FormHistory.historyType == "Labor Status Form")  # Filter by Labor Status history type
+        ))
+            .where(
+                (LaborStatusForm.studentSupervisee == self.student) &
+                (LaborStatusForm.startDate <= today) &
+                (LaborStatusForm.endDate >= today) &
+                (Department.isActive == True) &
+                (fn.BINARY(Department.DEPT_NAME) == "Labor Department") & 
```

Comment:
There's no department named "Labor Department", and the name might change soon anyway. We should probably use ACCOUNT and ORG so a name change doesn't break the query. Let's use ACCOUNT 6740 and ORG either 4022 or 4187.